primer/react
# bs/pageheader dynamic grip template areas

- initial
- make the examples as close as possible to dotcom usages
- remove console.log
- pair with mike
- lint
- remove console log
- Revert "pair with mike"
- "Address dom order issues - pair with mike"
- test(vrt): update snapshots
- seperate title area into its own components
- use grid to to keep the element visually in the same order but update dom order
- fix the grid issues on the title (paired with Mike)
- Use grid template areas
- fix failing snapshots and linting etc
- update expected styles in jest
- dynamicly render grid areas
- super WIP - still dynamicly render elements but based on if they intersect in the viewport. Becase we are dealing with display:none, just checking element is defined not enough



Closes #



### Changelog



#### New



#### Changed



#### Removed



### Rollout strategy




### Testing & Reviewing



### Merge checklist

- [ ] Integration tests pass at github/github ([Learn more about how to run integration tests](https://github.com/github/primer-engineering/blob/main/how-we-work/testing-primer-react-pr-at-dotcom.md))

Take a look at the [What we look for in reviews](https://github.com/primer/react/blob/main/contributor-docs/CONTRIBUTING.md#what-we-look-for-in-reviews) section of the contributing guidelines for more information on how we review PRs.
